proposal.rst

```diff
@@ -267,29 +285,29 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-* Weeks of bonding: get familiar with ranking algorithms, search for suitable datasets, assessment of the complexity of integrating large databases, and performance tests.
 
-In each next position assumes that testing and writing cross-platform part is included.
+* Weeks of bonding: Get more familiar with ranking algorithms, select specific suitable parts of the above datasets, discuss what is wrong with `#278 <https://github.com/xapian/xapian/pull/278>`_.
 
-* 1 Week - integrate datasets 
+* 1 Week - FIRE dataset integration. Each user have to download it by themself, because of 
+registration required. Implement training process using Letor API.  
```

Comment:
What will it mean to implement the training process? Do we have a parser for FIRE, or one close enough to adapt, or will you be writing something from scratch? What does "implement training process" mean here? The letor API provides all the hooks for doing this, so is this adding more functionality to xapian-evaluation, or some new tool?